pkg/query-service/rules/prom_rule_task.go

```diff
@@ -369,12 +369,11 @@ func (g *PromRuleTask) Eval(ctx context.Context, ts time.Time) {
 				rule.SetEvaluationTimestamp(t)
 			}(time.Now())
 
-			kvs := map[string]string{
-				"alertID": rule.ID(),
-				"source":  "alerts",
-				"client":  "query-service",
-			}
-			ctx = context.WithValue(ctx, common.LogCommentKey, kvs)
+			comment := ctxtypes.CommentFromContext(ctx)
+			comment.Set("alertID", rule.ID())
```

Comment:
change this to `alert_id` in consistency with `dashboardID` -> `dashboard_id` change?